cosmetics-web/app/controllers/responsible_persons/wizard/component_build_controller.rb

```diff
@@ -282,7 +282,7 @@ def update_contains_poisonous_ingredients
     end
 
     @component.update!(contains_poisonous_ingredients: params[:component][:contains_poisonous_ingredients])
-    if @component.contains_poisonous_ingredients?
+    if @component.contains_poisonous_ingredients? && !@component.notification.was_notified_before_eu_exit?
```

Comment:
Suggestion:
There is a method for the `!was_notified_before_eu_exit?`, seems clearer to read:
```suggestion
    if @component.contains_poisonous_ingredients? && @component.notification.was_notified_after_eu_exit?
```

Reply:
:thinking:  Did we even show the poisonous ingredients questions for "before EU Exit" case?

Reply:
No, but we introduced the poisonous ingredients question in this PR but want to skip the file upload, only allowing the file to be uploaded on the check your answers page. Not 100% sure on the reasoning for this but was part of the flow change requirements bruce which you can see in the comments on the linked jira ticket (there are a lot!)

Reply:
thanks for clarifying it. Missed those comments.